features/tab-tabstop-props.feature

```diff
@@ -14,6 +14,18 @@ Feature: Tab stop properties
       |    out    | -457200  |
 
 
+  @wip
+  Scenario Outline: Set tab stop position
+    Given a tab stop 0.5 inches <in-or-out> from the paragraph left edge
+     When I change the tab stop position to <position>
```

Comment:
The prior acceptance tests are a bit of a mixed bag on this because I refined my approach along the way. But as I came to see the "end-user" for this library as the developer using the API, I started using language in the acceptance tests that was more directly sensible to a developer rather than a typical "application" end-user.

So these have become: 'When I assign <value> to tab_stop.position' (or whatever the property is)

This makes them easy to write because the only thing that changes is the object.property part. It also makes them easier to make sense of in the steps .py file.